### ISSUE TITLE
Construct Validity and Psychometric Properties of the Tamil (India) Version of T-MoCA

### ISSUE THREAD

**Issue by ben-domingue** [ben-domingue/irw#282]
https://www.researchgate.net/profile/Mani-Abdul-Karim/publication/365373004_Construct_Validity_and_Psychometric_Properties_of_the_Tamil_India_Version_of_Montreal_Cognitive_Assessment_T-MoCA_in_Elderly/links/637315d254eb5f547cd3c715/Construct-Validity-and-Psychometric-Properties-of-the-Tamil-India-Version-of-Montreal-Cognitive-Assessment-T-MoCA-in-Elderly.pdf

https://osf.io/f7nkg/

**Comment by saviranadela** [comment]
yeah, it's good to split the files by different constructs. what's the difference between those sheets? @curious-frances
